tests/control_tasks/test_seq_labeling_ct.py

```diff
@@ -0,0 +1,120 @@
+import unittest
+from collections import Counter
+from cProfile import label
+from random import sample
+
+import neurox.control_tasks.seq_labeling_ct as ct
+import numpy as np
+
+# def create_seq_labeling_control_task_datasets(train_tokens, sample_from='same', dev_source=None, test_source=None):
+
+
+class TestCreateSeqLabelingControlTaskDatasets(unittest.TestCase):
+    def setUp(self):
+        self.three_label_data_skewed = {
+            "source": [
+                ["Fly", "me", "to", "the", "moon"],
+                ["I", "am", "walking", "on", "the", "moon"],
+                ["Now", "I", "am", "walking", "on", "a", "dream"],
+            ],
+            "target": [
+                ["A", "A", "A", "B", "A"],
+                ["A", "B", "A", "A", "B", "C"],
+                ["B", "A", "B", "A", "A", "B", "A"],
+            ],
+        }
+        self.two_labels_dataset_balanced = {
+            "source": [
+                ["I", "am", "walking", "on", "sunshine"],
+                "Tambourines",
+                "and",
+                "elephants",
+                "are",
+                "playin",
+                "in",
+                "the",
+                "band",
+            ],
+            "target": [
+                ["B", "A", "A", "B", "A"],
+                ["B", "A", "B", "A", "B", "A", "B", "A"],
+            ],
+        }
+        self.small_dataset_four_labels = {
+            "source": [["Is", "There", "life", "on", "Mars"]],
+            "target": [["A", "B", "C", "D", "A"]],
+        }
+
+        ints_list_of_lists = [[str(i)] for i in range(1000)]
+        modulo_seven_target = [["T"] if i % 7 == 0 else ["F"] for i in range(1000)]
+        self.modulo_seven_dataset = {
+            "source": ints_list_of_lists,
+            "target": modulo_seven_target,
+        }
+
+    def test_same_words_have_same_ct_label_in_same_dataset(self):
+        [ct_tokens] = ct.create_seq_labeling_control_task_datasets(
+            self.three_label_data_skewed
+        )
+        the_moon = (
+            ct_tokens["source"][0][3:] == ct_tokens["source"][1][4:]
+            and ct_tokens["target"][0][3:] == ct_tokens["target"][1][4:]
+        )
+        eye_am_walking_on = (
+            ct_tokens["source"][1][0:4] == ct_tokens["source"][2][1:5]
+            and ct_tokens["target"][1][0:4] == ct_tokens["target"][2][1:5]
+        )
+        assert the_moon and eye_am_walking_on
```

Comment:
Please use the unittest module's assert functions (https://docs.python.org/3/library/unittest.html#unittest.TestCase.debug)

You can have multiple asserts, so this one can become:
```python
self.assertEqual(ct_tokens["source"][1][0:4], ct_tokens["source"][2][1:5])
self.assertEqual(ct_tokens["target"][1][0:4], ct_tokens["target"][2][1:5])
...
```

This makes it easy to figure out when and why a test case is failing (and also helps test runners have better output)

Reply:
Done